resources/createdb.yaml

```diff
@@ -1,3 +1,44 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: stolon-createdb
+---
+apiVersion: rbac.authorization.k8s.io/v1alpha1
+kind: Role
+metadata:
+  name: stolon-createdb
+  namespace: default
+rules:
+  - apiGroups: [ "" ]
+    verbs:     [ "get" , "list" , "watch" ]
+    resources:
+      - secrets
+    resourceNames:
+      - "stolon"
+  - apiGroups: [ "" ]
+    verbs:     [ "get" , "list" , "watch" ]
+    resources:
+      - configmaps
+      - endpoints
+      - events
+      - pods
+      - persistentvolumes
+      - persistentvolumeclaims
+      - services
+---
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1alpha1
+metadata:
+  name: stolon-backup
```

Comment:
Is this supposed to reference the `stolon-createdb` Role (https://github.com/gravitational/stolon-app/pull/101/files#diff-fe264638d948263f726b3ed2550f650fR40)?